docs/contributor_guide/contributing_code.rst

```diff
@@ -23,8 +23,8 @@ Reductions
 Reduction constructors require a parameter corresponding to an estimator that
 implements the :code:`fit` method with the :code:`sample_weight` argument.
 Parity constraints for reductions are expressed via instances of various
-subclasses of the class :code:`fairlearn.reductions.Moment`. Formally,
-instances of the class :code:`Moment` implement vector-valued random variables
+subclasses of the class :class:`fairlearn.reductions.Moment`. Formally,
+instances of the class :class:`.Moment` implement vector-valued random variables
```

Comment:
as long as you've checked and these work, I'm happy with them :D

Reply:
I checked every single one yesterday because I made most of these months ago and didn't trust them 😅
It's just for splitting it over three lines to prevent the table from getting into a weird shape